### ISSUE TITLE
Messaging child bot crashes the process

### ISSUE THREAD

**Issue by HerrEurobeat** [HerrEurobeat/steam-comment-service-bot#46]
**Describe the bug**
Messaging a child bot crashes the node process

**Full error**
```
H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\src\bot.js:512
          bot.chatMessage(steamID, `This is one account running in a bot cluster.\nPlease add the main bot (Profile ID: ${new SteamID(controller.botobject[0].String(steamID)).getSteamID64()}) and send him a !help message.\nIf you want to check out what this is about, type: !about`)
                                                                                                                                                              ^

TypeError: controller.botobject[0].String is not a function
    at SteamUser.<anonymous> (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\src\bot.js:512:159)
    at SteamUser.emit (events.js:310:20)
    at SteamUser._emitIdEvent (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\utility.js:28:12)
    at SteamUser.<anonymous> (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\chatroom.js:61:8)
    at H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\classes\HandlerManager.js:37:12
    at Array.forEach (<anonymous>)
    at HandlerManager.emit (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\classes\HandlerManager.js:36:12)
    at SteamUser._handleMessage (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\messages.js:555:24)
    at SteamUser._handleNetMessage (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\messages.js:490:7)
    at SteamUser.processMulti (H:\Sonstiges\Steam\Steam Bots\steam-comment-service-bot\node_modules\steam-user\components\messages.js:582:9)
```

**Comment by HerrEurobeat** [comment]
Bug fixed, waiting for other bug fixes to be finished before pushing to 2.7.1 and applying fixed label.

**Comment by HerrEurobeat** [comment]
Fixed in commit 7b28378.
